docs/section22/Rule22-34.md

```diff
@@ -0,0 +1,38 @@
+
+# CHW&CW - Rule 22-34  
+
+**Rule ID:** 22-34  
+**Rule Description:** Baseline chilled water system that does not use purchased chilled water must only have no more than one CHW plant.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is not modeled with purchased chilled water.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is not modeled with purchased chilled water: `if Rule-18-8 == "NOT APPLICABLE":`
+
+## Rule Logic:  
+
+- For each chiller in B_RMR, save chiller to loop-chiller dictionary: `for chiller_b in B_RMR.ASHRAE229.chillers: loop_chiller_dict[chiller_b.cooling_loop].append(chiller_b)`
+
+**Rule Assertion - RMR:**
+
+- Case 1: If only one CHW plant is modeled: `if loop_chiller_dict.keys().length == 1: PASS`
+
+- Case 2: Else: `else: FAIL`
+
+**[Back](../_toc.md)**
+
+**Notes:**
+
+1. TBD if one loop per building, or change Case 2 to UNDETERMINED.
```

Comment:
I think case 2 should be Fail, and modeler would need to justify having more than one loop because in my opinion it conflicts App G language as it doesn't have any exceptions for multiple buildings scenario